### PR TITLE
Updated instructions for deploying code to testing and production.

### PR DESCRIPTION
**Pull request by critzo**
This PR updates the mlab-ns README so that it contains accurate information about how mlab-ns code is deployed to testing and production. @nkinkade PTAL?

---
This change is [Reviewable](https://reviewable.io/reviews/m-lab/mlab-ns/106)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,47 +3,17 @@ Status](https://travis-ci.org/m-lab/mlab-ns.svg?branch=master)](https://travis-c
 [![Coverage
 Status](https://coveralls.io/repos/m-lab/mlab-ns/badge.svg?branch=master&service=github)](https://coveralls.io/github/m-lab/mlab-ns?branch=master)
 
-# Deploying Code to mlab-ns
+# Deploying Code to mlab-nstesting (testing) and mlab-ns (production)
 
-To deploy code updates to mlab-ns or to deploy mlab-ns to a fresh GCP test project, follow the instructions below.
+mlab-ns code is built and pushed to a Google App Engine (GAE) service automatically by Travis.ci.
```
Comment (nkinkade) [inline README.md]: Small nit: s/Travis.ci/Travis CI/

Comment (critzo) [reply README.md]: Changed in my latest commit.

**Comment by critzo** [comment]
@nkinkade I tried deploying using gcloud commands to mlab-nstesting, but it did not work. Could you add a commit in a future PR to add instructions that work using gcloud commands?

**Comment by nkinkade** [comment]
@critzo: I can do that, but would you be able to revert the majority of this PR such that all the old instructions stay in place, such that I only have to update the specific references to `appcfg.py`?

**Comment by critzo** [comment]
@nkinkade I made another commit that adds the portion of the former instructions into a new section. If you could test and confirm this is working and update that section I'd be happy to test and review. Thanks.

**Comment by critzo** [comment]
@nkinkade PTAL?

**Pull request closed without merging**